### PR TITLE
Adds GroupViT to models exportable with ONNX

### PR DESCRIPTION
**Pull request by unography**
Adds GroupViT to models exportable with ONNX

### CODE REVIEW

**Comment by unography** [comment]
@regisss hopefully you don't have to make any changes this time to make the tests pass!

**Review by regisss — APPROVED** [review]
LGTM, you are on fire @unography :fire: 
Tests passed.

**Comment by regisss** [comment]
Pinging @sgugger for final approval

**Comment by LysandreJik** [comment]
Feel free to merge if you approve @lewtun

**Comment by regisss** [comment]
@lewtun Can you take a quick look at this PR and merge it when you approve? :slightly_smiling_face:

File: src/transformers/models/groupvit/configuration_groupvit.py [hunk 1]
```diff
@@ -343,3 +350,44 @@ def to_dict(self):
...
         return output
+
+
+class GroupViTOnnxConfig(OnnxConfig):
+    @property
+    def inputs(self) -> Mapping[str, Mapping[int, str]]:
+        return OrderedDict(
+            [
+                ("input_ids", {0: "batch", 1: "sequence"}),
+                ("pixel_values", {0: "batch"}),
```
Comment (lewtun) [inline src/transformers/models/groupvit/configuration_groupvit.py:361]: Same comment as in the OWLViT PR: should this be

```suggestion
                ("pixel_values", {0: "batch", 1: "num_channels", 2: "height", 3: "width"} }),
```

**Review by lewtun — APPROVED** [review]
Thanks for adding this model to the ONNX exporter @unography - as in the OWLViT case, I just have a small comment about the shape of `pixel_values`. Otherwise this looks great!

**Review by lewtun — APPROVED** [review]
Thanks a lot for iterating on the dynamic shapes @unography !

This LGTM, so gently pinging @sgugger for final approval

**Review by LysandreJik — APPROVED** [review]
LGTM, thank you @unography!

**Pull request merged**